other/materials_designer/workflows/formation_energy.ipynb

```
@@ -0,0 +1,670 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bank_materials_data = client.bank_materials.list(\n",
+    "    {\"tags\": \"elemental\", \"metadata.element\": {\"$in\": elements}},\n",
+    "    projection={\"limit\": len(elements)},\n",
+    ")\n",
+    "\n",
```

Ruff (0.15.20)

[error] `await` statement outside of a function

(F704)